py_modules/advanced_options.py

```diff
@@ -164,6 +166,17 @@ def get_default_options():
 
   options.append(max_tdp_on_resume)
 
+  limit_battery_charge = {
```

Comment:
this should be added to the ROG Ally advanced options method, not default_options.